tests/components/switch_as_x/test_config_flow.py

```diff
@@ -114,37 +117,88 @@ async def test_config_flow_registered_entity(
     assert config_entry.data == {}
     assert config_entry.options == {
         CONF_ENTITY_ID: "switch.ceiling",
+        CONF_INVERT: False,
         CONF_TARGET_DOMAIN: target_domain,
     }
 
     switch_entity_entry = registry.async_get("switch.ceiling")
     assert switch_entity_entry.hidden_by == hidden_by_after
 
 
+def get_suggested(schema, key):
```

Comment:
Isn't it possible to access the item by key in the schema? Then I'd just do that in the test instead of making this function that is only used once.

Reply:
Not that it's the *key* which has the suggestions, not the item, hence the sing-and-dance number

Reply:
Jag skulle istället för detta direkt i testet ha:

```py
schema_key = next(k for k in schema if k == key)
assert schema_key.description.get("suggested_value") is True
```

Reply:
You asked:
> Isn't it possible to access the item by key in the schema

That's not possible, but sure, the used-once-method can of course be inlined regardless of that 🤷